essentials.yml

```diff
@@ -1,6 +1,30 @@
 ---
+packaging:
+  config:
+    bundle:
+      groupId: "org.jenkins-ci.main"
+      artifactId: "jenkins-core-it"
+      vendor: "Jenkins project"
+      title: "Jenkins Integration Tests WAR"
+      description: "Jenkins WAR for integration tests"
+    war:
+      groupId: "org.jenkins-ci.main"
+      artifactId: "jenkins-war"
+      source:
+        dir: "."
 ath:
   useLocalSnapshots: false
   athRevision: "06caf67d37d945ec5228a80d73714fac692c28d3"
   categories:
     - org.jenkinsci.test.acceptance.junit.SmokeTest
+pct:
+  pct:
+    useLocalSnapshots: false
+    jth:
+      version: 2.38
+      passCustomJenkinsWAR: true
+    plugins:
+      - "ssh-slaves"
```

Comment:
I wonder if there is some reason to test this concrete set of plugins? Not against it, but would like to know why these and not others, and why is important to check that core is not introducing changes that break them